docs/cli/nrfu.md

```diff
@@ -16,10 +16,9 @@ ANTA provides a set of commands for performing NRFU tests on devices. These comm
 ### NRFU Command overview
 
 ```bash
-anta nrfu --help
 Usage: anta nrfu [OPTIONS] COMMAND [ARGS]...
 
-  Run NRFU against inventory devices
+  Run ANTA tests on devices.
```

Comment:
inventory was good here :p which devices?

Reply:
Just thought that it was confusing because it is not "all inventory devices" anymore. It depends on the CLI options.